rct229/rules/section6/section6rule3.py

```diff
@@ -0,0 +1,61 @@
+from rct229.data.schema_enums import schema_enums
+from rct229.data_fns.table_9_6_1_fns import table_9_6_1_lookup
+from rct229.rule_engine.rule_base import RuleDefinitionListIndexedBase, RuleDefinitionBase
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+
+GUEST_ROOM = schema_enums["LightingSpaceType2019ASHRAE901TG37"].GUEST_ROOM.name
+DORMITORY_LIVING_QUARTERS = schema_enums["LightingSpaceType2019ASHRAE901TG37"].DORMITORY_LIVING_QUARTERS.name
+DWELLING_UNITS = schema_enums["LightingSpaceType2019ASHRAE901TG37"].DWELLING_UNIT.name
+
+class Section6Rule3(RuleDefinitionListIndexedBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 6 (Lighting)"""
+
+    def __init__(self):
+        super(Section6Rule3, self).__init__(
+            rmrs_used=UserBaselineProposedVals(True, True, False),
```

Comment:
This rule uses user and proposed, so this should be:
`rmrs_used=UserBaselineProposedVals(True, False, True),`

Likewise for the nested rules.